src/js/common/schemaform/save-in-progress/RoutedSavableReviewPage.jsx

```diff
@@ -8,10 +8,20 @@ import SaveFormLink from '../save-in-progress/SaveFormLink';
 import SaveStatus from '../save-in-progress/SaveStatus';
 
 import { saveAndRedirectToReturnUrl, autoSaveForm, saveErrors } from './actions';
+import { getReviewPageOpenChapters } from '../state/selectors';
 import { toggleLoginModal } from '../../../login/actions';
 
 import { ReviewPage } from '../review/ReviewPage';
-import { setData, setPrivacyAgreement, setEditMode, setSubmission, submitForm, uploadFile } from '../actions';
+import {
+  closeReviewChapter,
+  openReviewChapter,
```

Comment:
This tripped me up for a little bit- couldn't figure out why the redux functions weren't firing on `ReviewPage`